Source/Scene/Scene.js

```diff
@@ -1668,7 +1668,7 @@ define([
             // TODO : Execute only opaque and translucent commands for now.
             var startPass = Pass.OPAQUE;
             var endPass = Pass.TRANSLUCENT;
-            for (var pass = startPass; pass < endPass; ++pass) {
+            for (var pass = startPass; pass <= endPass; ++pass) {
```

Comment:
This is for casting shadows in `executeShadowMapCommands` not `executeCommands` so its fine. For this pass, you don't need to worry about any of the OIT stuff since only depth is rendered.